example/example.js

```diff
@@ -0,0 +1,23 @@
+const { AudioProcessor } = require('../index'); // Assuming your index.js is in the parent directory
+
+async function runExample() {
+  const audioFilePath = '../audio/a.mp3';
+  const outputFilePath = './redacted_audio.wav';
+
+  try {
+    // Create a transcriber and a redactor from the factory
+    const transcriber = AudioProcessor.getTranscriber('deepgram',process.env.DEEPGRAM_API_KEY, audioFilePath);
+    const redactor = AudioProcessor.getRedactor('deepgram', audioFilePath);
```

Comment:
how come deepgram api key is not required here?

Reply:
For redaction of audio we dont need deepgram to do anything. Redactor just needs word level timestamps and words to be redacted.